Tests/GitHubRepositoryAutolinks.tests.ps1

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.Synopsis
+   Tests for GitHubRepositoryAutolink.ps1 module
+#>
+
+[CmdletBinding()]
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', '',
+    Justification='Suppress false positives in Pester code blocks')]
+param()
+
+# This is common test code setup logic for all Pester test files
+$moduleRootPath = Split-Path -Path $PSScriptRoot -Parent
+. (Join-Path -Path $moduleRootPath -ChildPath 'Tests\Common.ps1')
+
+try
+{
+    # Define Script-scoped, readonly, hidden variables.
+    @{
+        defaultRepoDesc = "This is a description."
+        defaultRepoHomePage = "https://www.microsoft.com/"
+        defaultRepoTopic = "microsoft"
+    }.GetEnumerator() | ForEach-Object {
+        Set-Variable -Force -Scope Script -Option ReadOnly -Visibility Private -Name $_.Key -Value $_.Value
+    }
+
+    Describe 'GitHubRepositoryAutolink\Get-GitHubRepositoryAutolink' {
+        BeforeAll {
+            $organizationName = $script:organizationName
+        }
+
+        Context 'When getting a GitHub Repository Autolinks by repository' {
+            BeforeAll {
+                $repoName = [Guid]::NewGuid().Guid
+
+                $repo = New-GitHubRepository -RepositoryName $repoName -OrganizationName $organizationName
+
+                $uriPrefix = 'PRJ-'
+                $urltemplate = 'https://company.issuetracker.com/browse/prj-<num>'
+                $isNumericOnly = $true
+
+                $newGitHubRepositoryAutolinkParms = @{
+                    OrganizationName = $organizationName
+                    UriPrefix = $uriPrefix
+                    UrlTemplate = $urltemplate
+                    IsNumericOnly = $isNumericOnly
+                }
+
+                New-GitHubRepositoryAutolink @newGitHubRepositoryAutolinkParms | Out-Null
+
+                $autoLinks = Get-GitHubRepositoryAutolink -OwnerName $organizationName -RepositoryName $repoName
+                $autolink = $autoLinks | Where-Object -Property AutolinkKeyPrefix -eq $uriPrefix
+            }
+
+            It 'Should have the expected type and additional properties' {
+                $team.PSObject.TypeNames[0] | Should -Be 'GitHub.RepositoryAutolink'
```

Comment:
This makes me feel like you didn't validate your tests before submitting the PR since this is referencing both a variable (`$team`) and properties (like `OranizationName`) that don't exist.